README.md

```diff
@@ -47,38 +45,72 @@ Then add Java to jenv for recognize it.
 ```sh
 jenv add $(/usr/libexec/java_home)
 ```
-</details>
-<details>
-<summary>3.a. Install SpringBoot (Optional)</summary>
+
+### 3.a. Install SpringBoot (Optional)
 
 Install Spring via homebrew
 
 ```sh
 brew install springboot
 ```
-</details>
-<details>
-<summary>3. Install Gradle</summary>
+
+### 3. Install Gradle
 
 Install Gradle and Gradle completion via homebrew
 
 ```sh
 brew install gradle gradle-completion
 ```
-For more visit [gradle-completion]https://github.com/gradle/gradle-completion repo on Github. You need to make your configurations for `bash` or `zsh`.
+For more visit [gradle-completion](https://github.com/gradle/gradle-completion) repo on Github. You need to make your configurations for `bash` or `zsh`.
 
-</details>
-<details>
-<summary>4. Now run the project with...</summary>
+### 4. Now run the project with...
 ```sh
 gradle bootRun
 ```
 or
 ```sh
 ./gradlew bootRun
-``**
-</details>
+```
 
 ## For Windows
 
-**TBD***
+First clone this project to your machine.
+git clone git@github.com:gungorkocak/emeq-counter.git && cd emeq-counter
```

Comment:
You can use back quote to make the codes to be written on the terminal more obvious. Like that
```
codes come here
```